dashboard/app/models/user.rb

```diff
@@ -248,6 +248,8 @@ def delete_course_as_facilitator(course)
   end
 
   def log_admin_save
+    return if new_record? && !admin?
```

Comment:
Good point. We should be consistent, rather than splitting up the logic. I'll move it to the callback